code/game/machinery/doors/brigdoors.dm

```diff
@@ -118,34 +114,37 @@
 			return
 	atom_say("[src] beeps, \"[occupant]: [notifytext]\"")
 
-/obj/machinery/door_timer/Initialize()
-	..()
+/obj/machinery/door_timer/Initialize(mapload)
+	. = ..()
```

Comment:
```suggestion
	..()
```
No need to take the return type here since you'll return `INITIALIZE_HINT_LATELOAD` later